plugins/syncthing/strelaysrv_

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env sh
```

Comment:
Please use `/bin/sh`.
The `env` approach is used for *configurable* interpreters (e.g. python or perl).